### DISCUSSION TITLE
Question on bandwidth and Span ports

### DISCUSSION THREAD

**Discussion by fabiopiccioni** [Security-Onion-Solutions/securityonion#13097] [Unsupported Versions] [answered]
Hello everybody.
Maybe i'm Posting a stupid question about Traffic Monitoring.
If i have a manged switch with multiple vlans to separate networks, let's say 6 ports, and I span all traffic to a single physical porta to monitor with SO sensor, what happens if all ports will be used at maximum throughout available? Monitor port will only manage 1gb/s.

Thank you for any link I could use to find some information.

**Comment by dougburks** [comment, marked as answer]
If the span port is a 1 gigabit port, then any traffic beyond 1 gigabit will not be copied to the span port. 

For additional information, please check the documentation for your specific switch or reach out your switch vendor.
